src/Transport/Topology/Topologies/ForwardingTopologySectionManager.cs

```diff
@@ -130,14 +130,25 @@ public TopologySectionInternal DetermineSendDestination(string destination)
                         {
                             namespaces = new[]
                             {
-                                new RuntimeNamespaceInfo(configured.Alias, configured.Connection, configured.Purpose, NamespaceMode.Active)
+                                new RuntimeNamespaceInfo(configured.Alias, configured.Connection, configured.Purpose)
                             };
                         }
                     }
                 }
-                else // sending to the partition
+                else
                 {
-                    namespaces = namespacePartitioningStrategy.GetNamespaces(PartitioningIntent.Sending).ToArray();
+                    var configured = namespaceConfigurations.FirstOrDefault(n => n.RegisteredEndpoints.Contains(d, StringComparer.OrdinalIgnoreCase));
```

Comment:
`configuredToRouteToAnotherNamespace`? everything is configured at some point 😃

Reply:
will change